src/openrct2/world/map.h

```diff
@@ -334,6 +334,8 @@ enum {
     MAP_SELECT_TYPE_EDGE_2,
     MAP_SELECT_TYPE_EDGE_3,
 };
+// Transformation edge to first of connected corners
```

Comment:
I don't understand what this comment is trying to say, or what this define is for. Could you clarify?

Reply:
Each to pairs of corners create a edge. This define is an offset used to extract corner back from the given edge. This corner and the connected corner (2 of them make that edge) is then used for topography calculations.

When I look at it now, perhaps it would be better to keep it in the c file only as it is not shared between sources.